Remotion/Web/Core/UI/Controls/Badge.cs

```diff
@@ -69,9 +72,11 @@ public string Value
     [PersistenceMode (PersistenceMode.Attribute)]
     [DefaultValue ("")]
     [NotifyParentProperty (true)]
+    [AllowNull]
```

Comment:
```suggestion
```

Reply:
Nominally, that's correc.t it's just that in Badge, we already made the value non-nullable, just used a fallback. so, it just felt cleaner this way.